terraform/03_gke_cluster.tf

```diff
@@ -157,7 +157,7 @@ resource "null_resource" "deploy_services" {
 # For the purposes of Sandbox, we can mitigate by waiting a few sec to ensure kubectl apply completes
```

Comment:
@simonz130 do you have the context for this comment? Is there a reason we can't `kubectl wait`?

Reply:
I think we should kubectl wait :)